protos/action/action.proto

```diff
@@ -177,6 +183,26 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+enum OrbitYawBehavior {
+        ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER = 0;
+        ORBIT_YAW_BEHAVIOUR_HOLD_INITIAL_HEADING = 1;
+        ORBIT_YAW_BEHAVIOUR_UNCONTROLLED = 2;
+        ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TANGENT_TO_CIRCLE = 3;
+        ORBIT_YAW_BEHAVIOUR_RC_CONTROLLED = 4;
+}
+    
+message DoOrbitRequest {    
+    float radius_m = 1; // Radius of circle (in meters)
+    float velocity_ms = 2; // Tangential velocity (in m/s)
+    OrbitYawBehavior yaw_behavior = 3; // Yaw behavior of vehicle (ORBIT_YAW_BEHAVIOUR)
+    double latitude_deg = 5; // Center point latitude in degrees. NAN: current center
+    double longitude_deg = 6; // Center point longitude in degrees. NAN: current center
+    double absolute_altitude_m = 7; // Center point altitude in meters. NAN: current elevation
```

Comment:
Let's default them to NaN, then :blush:, if you agree.

This requires you to add `import "mavsdk_options.proto";` at the top of `action.proto` (see e.g. [here](https://github.com/mavlink/MAVSDK-Proto/blob/main/protos/mission/mission.proto#L5))

```suggestion
    double latitude_deg = 5 [(mavsdk.options.default_value)="NaN"; // Center point latitude in degrees. NAN: use current latitude for center
    double longitude_deg = 6 [(mavsdk.options.default_value)="NaN"; // Center point longitude in degrees. NAN: use current longitude for center
    double absolute_altitude_m = 7 [(mavsdk.options.default_value)="NaN"; // Center point altitude in meters. NAN: use current altitude for center
```